docs/web3-eth.rst

```diff
@@ -2015,11 +2015,11 @@ Example
 
 .. code-block:: javascript
 
-    web3.eth.submitWork([
+    web3.eth.submitWork(
```

Comment:
It seems the method should accept an array of strings:
https://github.com/ChainSafe/web3.js/blob/8d369a9e76267df9ce666283521c1fd6d3f77fc4/packages/web3-eth/types/index.d.ts#L388
```js
    submitWork(
        data: [string, string, string],
        callback?: (error: Error, result: boolean) => void
    ): Promise<boolean>;
```


And here are also usage:

- https://vscode.dev/github/ChainSafe/web3.js/blob/SriCharanGundla-fix-typo/packages/web3-eth/types/tests/eth.tests.ts#L583
- https://vscode.dev/github/ChainSafe/web3.js/blob/SriCharanGundla-fix-typo/packages/web3-eth/types/tests/eth.tests.ts#L590

Did you miss updating related stuff? Or do I miss something here?

Reply:
@Muhammad-Altabba submit work expects 3 params: 
https://github.com/web3/web3.js/blob/47b9769d99c70ea985a5e639c9ca37ebb5c7cde9/packages/web3-eth/src/index.js#L564

passing array gives:
```
/node_modules/web3-core-method/lib/index.js:113
        throw errors.InvalidNumberOfParams(args.length, this.params, this.name);
        ^

Error: Invalid number of parameters for "submitWork". Got 1 expected 3!

```
In types file link you shared , types are not updated. cc: @luu-alex these also needs update.

Reply:
You are correct, thank you @Muhammad-Altabba !